elixir-1.17.yaml

```diff
@@ -19,8 +19,9 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - erlang
-      - erlang-dev
+      # compile Elixir with older supported version of Erlang. Reason: if we compile with the latest version of Erlang, the compiled Elixir will not work with the older Erlang versions.
```

Comment:
Is upstream doing the same?

Reply:
In the upstream it is not specify It says that we have to use "At least Erlang/OTP 25.0 is required to build Elixir"

I have created an issue about the problem we were facing that could help to understand why we did these changes.
https://github.com/chainguard-dev/enterprise-packages/issues/6111